src/components/Toggle/index.js

```diff
@@ -34,16 +34,14 @@ class Toggle extends Component {
 
     return (
       <div className='w5 relative flex items-center br-pill ba b--light-navy bg-dark-navy'>
-        <label className={labelClasses} htmlFor='left'>
+        <label className={labelClasses} htmlFor={`left-${leftOption}`}>
```

Comment:
Do we need to do something to handle any spaces that may be in the `leftOption` ?

`${leftOption.replace(' ', '-')}`